simulator/customer_abstract.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, unique_id, transaction_model, fraudster):
         super().__init__(unique_id, transaction_model)
 
         # copy parameters from model
-        self.params = copy.deepcopy(self.model.parameters)
+        self.params = self.model.parameters
```

Comment:
Sorry should've written my comment in the pull request I guess... so again: I think the deep copy is necessary so that individual clients cannot mess with the model's parameters, which in turn would influence the other customers. Took me forever to find out that this was happening (since it only led to slightly weird behaviour that I couldn't really explain).

Reply:
The problem doesn't seem to exist anymore; removing deepcopy seems fine.